Update sys-dm-fts-parser-transact-sql.md (#8705)
MicrosoftDocs/sql-docs: docs/relational-databases/system-dynamic-management-views/sys-dm-fts-parser-transact-sql.md

  
## Permissions  

>  Requires membership in the **sysadmin** fixed server role and access rights to the specified stoplist.  

Reviewer: Please make the requisite change to this line.

For example:

```suggestion
Requires CREATE FULLTEXT CATALOG permission, and access rights to the specified stoplist.
```

>  

Reviewer: ```suggestion
```

>  Verified that the "create fulltext catalog" permission is the bare minimum permission needed.

Reviewer: ```suggestion
```
